src/panels/lovelace/cards/energy/hui-energy-distribution-card.ts

```diff
@@ -81,8 +83,9 @@ class HuiEnergyDistrubutionCard
     const hasConsumption = true;
 
     const hasSolarProduction = types.solar !== undefined;
-    const hasBattery = types.battery !== undefined;
+    const hasBattery = false; // types.battery !== undefined;
```

Comment:
```suggestion
    const hasBattery = types.battery !== undefined;
```
I think we must revert this one.

Reply:
you think 😄